roles/init_dbserver/tasks/pg_initdb.yml

```diff
@@ -32,6 +32,15 @@
   when: ansible_os_family == 'Debian'
   become: true
 
+- name: Add EDB Postgres Tuner to shared_preload_libraries
+  ansible.builtin.set_fact:
+    pg_shared_libraries_list: "{{ pg_shared_libraries_list + \
```

Comment:
Same as above. You don't need to set the parameter here. If we include it in the postgresql.conf.template with the condition, it will be less expensive and will be better there.

Reply:
Ok, will do.